### PR TITLE
Admin tabs, Mailing Scheduler, Dropdown Menu all completed

### PR DESCRIPTION
**Pull request by adnanhemani**

### CODE REVIEW

File: app/views/shared/_head.html.erb [hunk 1]
```diff
@@ -26,8 +25,13 @@
               </li>
             <% else %>
               <% if session[:is_admin].nil? %>
+              <li><a href='/'>Home</a></li>
                 <li><%= link_to "Edit My Info", edit_user_path(session[:user_id]), {method: 'get'} %></li>
               <% else %>
+                  <li><%= link_to "All", admin_show_all_path(session[:user_id]), {method: 'get'} %></li>
```
Comment (dasolyoon) [inline app/views/shared/_head.html.erb]: Not a priority, but can you actually put these inside a different navbar/area (and especially not in the shared file)? Maybe in admins/show instead--it's not that intuitive to go look at the overall navbar to filter teams

File: app/controllers/admins_controller.rb [hunk 2]
```diff
@@ -30,9 +30,30 @@ def update
   end
 
   def show
```
Comment (dasolyoon) [inline app/controllers/admins_controller.rb]: rename to "show_all" for consistency

File: app/controllers/admins_controller.rb [hunk 3]
```diff
@@ -30,9 +30,30 @@ def update
...
+    render 'show'
+  end
+  
+  def show_pending
+    @stat_loc = "Team List: Showing Pending Teams"
+    @teams_li = Team.where("approved = ? AND submitted = ?", false, true)
+    render 'show'
+  end
+  
+  def forming
```
Comment (dasolyoon) [inline app/controllers/admins_controller.rb]: rename to "show_forming"

File: app/views/shared/_head.html.erb [hunk 4]
```diff
@@ -26,8 +25,13 @@
               </li>
             <% else %>
               <% if session[:is_admin].nil? %>
+              <li><a href='/'>Home</a></li>
```
Comment (dasolyoon) [inline app/views/shared/_head.html.erb]: Can you move this back up to line 18? (before the line that says      <% if session[:user_id].nil? %> )

File: features/admin_edit_team.feature [hunk 5]
```diff
@@ -20,22 +20,22 @@ Feature: An admin edits a team
...
+    Then pending: I should see "Removed Jorge from team."
+    And pending: the team with passcode "penguindrool" should not be submitted
 
   Scenario: An admin successfully removes a user from an approved team
-    Given the team with passcode "penguindrool" is approved with discussion number "54321"
-    When I press "remove_Jorge"
-    Then I should see "Removed Jorge from team."
-    And the team with passcode "penguindrool" should not be approved
-    And the team with passcode "penguindrool" should not be submitted
+    Given pending: the team with passcode "penguindrool" is approved with discussion number "54321"
```
Comment (dasolyoon) [inline features/admin_edit_team.feature]: just make these pass first

File: app/controllers/admins_controller.rb (hunk 2 above)
Comment (SidMasih) [inline app/controllers/admins_controller.rb]: fixing

Comment (SidMasih) [inline app/controllers/admins_controller.rb]: fixed

File: app/views/shared/_head.html.erb (hunk 4 above)
Comment (SidMasih) [inline app/views/shared/_head.html.erb]: done

File: features/admin_edit_team.feature (hunk 5 above)
Comment (SidMasih) [inline features/admin_edit_team.feature]: will do today

**Pull request merged**